test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -114,7 +112,10 @@ def func(x):
 
         _run_test_with_fx_to_onnx_exporter_and_onnx_runtime(func, (tensor_x,))
 
-    def test_func_with_args_and_kwargs(self):
+    # AssertionError: Dynamo input/output is not consistent with traced input/output
+    # https://github.com/pytorch/pytorch/issues/96379
+    @unittest.expectedFailure
```

Comment:
@wschin This is an issue in `dynamo.export`, I have opened an issue. Although it is bad practice to use mutable defaults, I think dynamo needs to have a clearly defined behavior in response to that, instead of raising seemingly unrelated error.

Is this an example from a real model?

I added another version of this test with optional python number. That one works with dynamo but fails with onnxscript.

Reply:
It's simple python code, so I consider it `real`. Please fix this test before merging. We need to try the best not to disable any tests as we don't have enough tests.